### DISCUSSION TITLE
Error after build: Failed to process build result for "chunks"

### DISCUSSION THREAD

**Discussion by kindlyfire** [vercel/community#1281] [Help] [answered]
### Summary

Hello,

My new project is getting an error after build. It seems to me like the app build process itself succeeds, but the error happens in the Vercel code that runs after and bundles the built files.

My first build succeeded, with the end of the logs being:

```
08:26:47 PM [build] 2 page(s) built in 2.63s
08:26:47 PM [build] Complete!
Done in 4.91s.
Build Completed in /vercel/output [20s]
Generated build outputs:
 - Static files: 34
 - Serverless Functions: 0
 - Edge Functions: 0
Deployed outputs in 1s
Build completed. Populating build cache...
Uploading build cache [20.43 MB]...
Build cache uploaded: 1.495s
```

After a commit, it no longer builds, with the following error:

```
09:49:25 PM [build] 2 page(s) built in 2.15s
09:49:25 PM [build] Complete!
Done in 3.86s.
Build Completed in /vercel/output [15s]
Failed to process build result for "chunks". Data: {"type":"FileFsRef"}.
[Error: EISDIR: illegal operation on a directory, read] {
  errno: -21,
  code: 'EISDIR',
  syscall: 'read'
}
```

The problem is, this error happens even when I try to re-deploy the initial commit that previously deployed correctly. I do not remember changing any settings in the project. It's a simple Astro project.

Deployment IDs:
- Succeeded: `2YeGLQv7mDcDZpZMseVux4yPWqtM`
- Redeploy of the succeeded one, which failed: `EGNqnDtqiPdo8Y1JRgYVkd1rPRGx`

**Comment by adrileo3** [comment]
Same thing happening here.

**Comment by ferdinandalexa** [comment]
I had same issue with Astro Project too. **I tried to redeploy few minutes ago and all works now**.

**Reply by adrileo3** [reply]
can confirm works now.

**Comment by swarnava** [comment, marked as answer]
We have pushed a fix for this few hours ago. Are you still facing this issue?

**Reply by kindlyfire** [reply]
No, everything works now. Thanks!
